testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -136,6 +136,17 @@ TestNodeProxy : UnitTest {
 
 	}
 
+	test_synthDefControl_build {
+		var s = Server.default;
+		s.quit;
+		this.bootServer(s);
+		s.sync;
+		1.wait;
+
+		Ndef.clear;
+		Ndef(\x, { Silent.ar });
+		this.assert(Ndef(\x).objects.first.hasFadeTimeControl, "functions should register their fadeTime control");
```

Comment:
fixed

Reply:
why is the `1.wait` after `s.sync` necessary (here and in the test above)? if it is please leave a comment explaining why, otherwise remove it. `1.01.wait` is there to give the Ndef time to release I'm guessing.

Also, how does `Ndef` know which server to use? Should there be additional modifications now that `s` is not `Server.default`?

Reply:
ah! Sorry, copy and paste. Fixed just now.